madmom/features/tempo.py

```diff
@@ -764,14 +960,23 @@ class DBNTempoEstimationProcessor(BaseTempoEstimationProcessor):
     MIN_BPM = 40.
     MAX_BPM = 250.
     HIST_SMOOTH = 9
-    ACT_SMOOTH = 0
+    ACT_SMOOTH = 0.
 
     def __init__(self, min_bpm=MIN_BPM, max_bpm=MAX_BPM, act_smooth=ACT_SMOOTH,
-                 hist_smooth=HIST_SMOOTH, fps=None, **kwargs):
+                 hist_smooth=HIST_SMOOTH, fps=None, online=False, **kwargs):
         # pylint: disable=unused-argument
         super(DBNTempoEstimationProcessor, self).__init__(
             min_bpm=min_bpm, max_bpm=max_bpm, act_smooth=act_smooth,
-            hist_smooth=hist_smooth, fps=fps, **kwargs)
+            hist_smooth=hist_smooth, fps=fps, online=online, **kwargs)
+        # save additional variables
+        if self.online:
+            self._dbn = DBNBeatTrackingProcessor(min_bpm=self.min_bpm,
+                                                 max_bpm=self.max_bpm,
+                                                 fps=self.fps)
```

Comment:
I would create the `DBNBeatTrackingProcessor` as `self.dbn` not only in online, but also in offline mode and use change `interval_histogram()` accordingly. While doing this, it would be good to also pass `**kwargs` instead of setting `num_tempi=None`.

The `from .beats import DBNBeatTrackingProcessor` should go in here rather than at the module level.